frontend/ui/src/components/kanban-board/index.tsx

```
@@ -1,351 +1,188 @@
-import type { ColumnWithTasks, NestedColumns, Task } from "@incmix/utils/types"
-import { Card, Flex, Spinner } from "@radix-ui/themes"
-import { useCallback, useEffect, useMemo, useRef, useState } from "react"
-import invariant from "tiny-invariant"
-import { CardContent } from "../card/card"
-import { Board } from "./board"
-import { BoardContext, type BoardContextValue } from "./board-context"
 
-export type ColumnMap = { [columnId: string]: ColumnWithTasks }
+import { Board } from "./board";
+import { KanbanImages } from "./images";
+import { TCard } from "./types";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)